### PR TITLE
Update copyright and file headers

### PR DESCRIPTION
**Pull request by mliu49**
This PR updates the copyright date in the LICENSE.txt file. It also adds a new script to automatically update all file headers based on the LICENSE.txt file and updates the headers with that script.

To review, check the diffs to make sure no important lines were accidentally lost.

Small note: Some manual changes were needed before I ran the script the first time since the file headers were not uniform. The script as it is now works properly with the updated files, so updating headers in the future should be pretty painless.

### CODE REVIEW

File: rmgpy/tools/__init__.py [hunk 1]
```diff
@@ -1,29 +1,3 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
```
Comment (amarkpayne) [inline rmgpy/tools/__init__.py]: License missing

File: rmgpy/transport.py [hunk 2]
```diff
@@ -1,3 +1,33 @@
...
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE #
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER      #
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING     #
+# FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER         #
+# DEALINGS IN THE SOFTWARE.                                                   #
+#                                                                             #
+###############################################################################
+
 '''
 Created on Feb 26, 2013
```
Comment (amarkpayne) [inline rmgpy/transport.py]: Should we keep the comment below this line?

File: rmgpy/version.py [hunk 3]
```diff
@@ -1,3 +1,31 @@
-# This file describes the version of RMG-Py
```
Comment (amarkpayne) [inline rmgpy/version.py]: This line gets removed accidently

**Comment by mliu49** [comment]
I removed the author tags as we discussed. Let me know when you've reviewed the new commits, and then I'll rebase and squash.

**Pull request merged**